### PR TITLE
Added GBA Emulator

### PR DESCRIPTION
**Pull request by ahmed-shehata**
Added GBA Emulator + TSLint Settings

### CODE REVIEW

File: src/components/Apps/index.tsx [hunk 1]
```diff
@@ -1,76 +1,90 @@
...
 
+const gbaId = genId()
+const gba = Map({
+  id: gbaId,
+  Component: pure(() => <iframe
+    style={{width: "100%", height: "100%", border: 0}}
+    src="http://endrift.github.io/gbajs/"
+    />
+  ),
+  icon: <HextrisIcon/>,
```
Comment (ahmed-shehata) [inline src/components/Apps/index.tsx]: Need to get a GBA icon

**Comment by aghoneim92** [comment]
😍 loving this. Thanks for adding linting too!

**Comment by aghoneim92** [comment]
@ahmed-shehata It appears that you've changed all single quotes to double. Guessing you added an extension that automatically does that or tslint rule ;)? Please revert those changes :)

**Comment by aghoneim92** [comment]
Small tip: it's recommended to use the infinitive in commit messages / prs / issues
ex: Add GBA Emulator.

**Pull request merged**